apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue

```diff
@@ -212,14 +253,20 @@ const listCostAnalysisTableData = async (): Promise<CostAnalyzeResponse<CostAnal
             data_source_id: costAnalysisPageStore.selectedDataSourceId,
             query: {
                 granularity: costAnalysisPageState.granularity,
-                group_by: costAnalysisPageState.groupBy,
+                group_by: [...costAnalysisPageState.groupBy, 'usage_unit'],
```

Comment:
nitpick: It seems we don't need `usage_unit` when there's no `usage_type` in groupBy.